test/fixtures/unit.js

```diff
@@ -11,6 +11,27 @@ function copyErrors (testEnd) {
   return testEnd;
 }
 
+/**
+ * Creates an object that has a cyclical reference.
+ */
+function createCyclical () {
+  const cyclical = {};
+  cyclical.cycle = cyclical;
```

Comment:
In order to excercise the stack a bit bit, perhaps add one layer of indirection here, e.g. `cyclical = { sub: {}; cyclical.sub.cycle = cyclical;`.

Reply:
Added an additional structure with a subobject cycle.